Build failure with missing symbol error

Failed Project:Falcon Oozie Adaptor

Error:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project falcon-oozie-adaptor: Compilation failure: Compilation failure:
[ERROR] /ws/HDP-falcon/oozie/src/main/java/org/apache/falcon/oozie/OozieOrchestrationWorkflowBuilder.java:[283,19] error: cannot find symbol
[ERROR] variable lib of type FileStatus
[ERROR] /ws/HDP-falcon/oozie/src/main/java/org/apache/falcon/service/SharedLibraryHostingService.java:[83,14] error: cannot find symbol

--- a/pom.xml
+++ b/pom.xml
@@ -803,6 +803,10 @@
                         <artifactId>hadoop-auth</artifactId>
                     </exclusion>
                     <exclusion>
+                        <groupId>org.apache.hadoop</groupId>
+                        <artifactId>hadoop-core</artifactId>
+                    </exclusion>
+                    <exclusion>
                         <groupId>org.apache.activemq</groupId>
                         <artifactId>activemq-client</artifactId>
                     </exclusion>

